src/formats/templates/swift/class-members.template

```diff
@@ -0,0 +1,30 @@
+<%
+//
+// Copyright 2019 Alaska Air Group or its affiliates. All Rights Reserved.
```

Comment:
Is this intentional? (this file is actually code from another org?)

Reply:
Ah yeah I should add a comment that this is based on [any.swift.template](https://github.com/amzn/style-dictionary/blob/main/lib/common/templates/ios-swift/any.swift.template) from the style dictionary source with a small tweak to remove the `static` keyword.

Germain actually opened a [PR](https://github.com/amzn/style-dictionary/pull/972) to handle this upstream but so far it hasn't been reviewed.